proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -218,7 +218,15 @@ service Microgrid {
     returns (AddComponentInclusionBoundsResponse);
 
   // Sets the active power output of a component with a given ID, provided the
-  // component supports it.
+  // component supports it. The power output is specified in watts.
+  //
+  // The power output can be -ve or +ve, depending on whether the component is
+  // supposed to be discharging or charging, respectively.
+  //
+  // The return value is the timestamp until which the given power command will
+  // stay in effect. After this timestamp, the component's active power will be
+  // set to 0, if the API receives no further command to change it before then.
+  // Typically, this timestamp will be set to the current time plus 60 seconds.
```

Comment:
Typically is not a very strong definition. I'd suggest to refer to a default value if it exists or write that this is implementation specific but typically (or usually), ...

Reply:
I changed it to "By default". This needs to be guaranteed in the implementation.